Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchable.cs

```diff
@@ -78,6 +78,7 @@ public Vector3 LocalRight
         /// </summary>
```

Comment:
The summary should be moved down to decorate `public Collider` not the private member

Reply:
fixed